packages/forms/resources/views/components/select.blade.php

```diff
@@ -74,6 +75,7 @@ class="fi-fo-select"
                         <option
                             @disabled($isOptionDisabled($value, $label))
                             value="{{ $value }}"
+                            wire:key="{{ $value }}"
```

Comment:
Pretty sure this would have to be scoped by the component state path.
Will leave this to @danharrin for a proper review.